test/controllers/email/VerifyPasscodeControllerSpec.scala

```diff
@@ -499,10 +514,19 @@ class VerifyPasscodeControllerSpec extends ControllerBaseSpec with MockEmailVeri
 
       "there is an email in session" should {
 
-        lazy val result = TestVerifyPasscodeController.contactPrefSendVerification()(paperRequestWithEmail)
+        lazy val result = {
```

Comment:
Make sure to test all branches of code when the `contactPrefSendVerification` action is reworked. It's missing some now but no point focusing on that until the change.